guidelines/content-guidelines/style-and-standards.md

```diff
@@ -17,7 +17,7 @@ Use active voice whenever possible. Active voice is clear, concise, and it avoid
 
 There are different tones for different types of technical documentation. The documentation can range from instructional to somewhat conversational, but always with the goal of helping users understand how to use the product for practical purposes and, in blogs and release notes, also helping business users understand changes. While writing Kyma documentation, use semi-formal style and imperative mood. The imperative mood tells the reader directly to do something. Use the imperative mood to write instructional documentation such as procedures and tutorials. Other moods can imply optional behavior.
 
-> **NOTE:** Avoid using unnecessary words such as "please" or "remember".
+> **NOTE:** Avoid using unnecessary words such as "please" or "remember."
```

Comment:
Wrong interpunction order. " closes the word or sequence you refer to and a dot closes the whole sentence.
Please adjust this comment to the rest of the document.